CSharpSourceCode/Battle/CrosshairMissionBehavior/CustomCrosshairMissionBehavior.cs

```diff
@@ -100,7 +100,7 @@ private void InitializeCrosshairs()
             _weaponCrosshair.InitializeCrosshair();
             _sniperScope = new SniperScope();
 
-            if (Agent.Main.IsAbilityUser() && (_abilityComponent = Agent.Main.GetComponent<AbilityComponent>()) != null)
```

Comment:
Is there any particulat use case for removing this as well? Or just to avoid a double check. I am always hesitant to remove double checks, they don't bother anyone and who knows what it might protect against.

Reply:
Yeah, sometimes it's better to check twice, but there we have more. Here are all places we had or still have IsAbilityUser checks:
CustomCrosshairMissionBehavior:
1) CustomCrosshairMissionBehavior.OnMissionScreenTick() - to check if ability crosshair should be shown;
2) AbilityManagerMissionLogic.HandleInput() - to check if player agent should run its extension CastCurrentAbility(), but HandleInput method also checks if current ability crosshair is visible what mean
3) Ability.CanCast() aslo to check if caster can use ability;

Reply:
i think here we can remove this one. It's nothing to worry about if crosshairs are initialized, but player is not an ability user, because to see crosshair you anyway pass check during OnMissionScreenTick()

Reply:
as discussed, agreed. resolving